js/layout-javascript/feed-comments-code.js

```diff
@@ -1153,23 +1204,20 @@ DynamicList.prototype.getCommentsCount = function(element) {
 
 DynamicList.prototype.connectToCommentsDataSource = function(id) {
   var _this = this;
-  var options = {
-    offline: true
+  var content = {
+    contentDataSourceEntryId: id,
+    type: 'comment'
   }
-
-  return Fliplet.DataSources.connect(_this.data.commentsDataSourceId, options)
-    .then(function(connection) {
-      _this.commentsConnection.push({
-        contentDataSourceEntryId: id,
-        connection: connection
-      });
-      return connection.query({
+  return Fliplet.Content({dataSourceId: _this.data.commentsDataSourceId})
```

Comment:
@hcarneiro Why does this use `Fliplet.Content` but not `Fliplet.Profile.Content`?

`Fliplet.Profile.Content` automatically adds the user's information to the contents created.